modules/test/playwright/tests/notification-web/userNotification.spec.ts

```diff
@@ -134,3 +134,62 @@ test('review comment is added to user notification', async ({
 		})
 	).toBeVisible();
 });
+
+test('approve or reject modal appear even after doing a comment on the comments section', async ({
```

Comment:
Hey @rebsilva, I think we can move this test to the portal-workflow-task-web module since what we are trying to do is approve or reject a workflow task, what do you think about that?

Reply:
true! i will put it inside the `portal-workflow-task-web/assignments.spec.ts` file then :cherry_blossom:

Reply:
actually i will create another file called `comments.spec.ts` since its the focus here